### PR TITLE
Asking for a code review for the Restaurant Page project

### PR DESCRIPTION
**Pull request by stratospherique**

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+linters:
+  eslint:
+    config: './eslint.config'
```
Comment (dipto0321) [inline .stickler.yml]: Change this path like this 👇 
```yaml
linters:
  eslint:
    config: './.eslintrc.json'
```

**Review by dipto0321 — APPROVED** [review]
Hi @stratospherique ,

Good job :clap: ! Nice UI. This project successfully met the main requirements. Before merging this project please delete that `eslint.config` file and change path in `.stickler.yml` (Check my comment 👇 ) and then fix all the linter issues if there any. This is from me 👍 

Thanks 🙏

File: .stickler.yml (hunk 1 above)
Comment (stratospherique) [reply .stickler.yml]: - I followed your suggestion and I still have 3 non-code related errors. Should I change the linting style from **Airnb** to **standard** ?

**Comment by stratospherique** [comment]
### All checks have passed :tada: :tada: :tada: .

**Pull request merged**